repository/pom.xml

```diff
@@ -309,6 +309,12 @@
             <artifactId>java-statsd-client</artifactId>
             <version>3.1.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.atlas</groupId>
+            <artifactId>atlas-graphdb-janus</artifactId>
+            <version>3.0.0-SNAPSHOT</version>
```

Comment:
nit: Move version to `<properties>` section

Reply:
added at project level now